conda/utils.py

```diff
@@ -255,8 +255,16 @@ def find_parent_shell(path=False):
         #          "To proceed, please conda install psutil")
         return None
```

Comment:
Let's go with it for now.  We should add a test for that behavior sometime very soon (but perhaps after the release) so that we understand the failure mode.